RecoTracker/FinalTrackSelectors/plugins/getBestVertex.h

```diff
@@ -2,11 +2,9 @@
 #include "DataFormats/VertexReco/interface/Vertex.h"
 #include <limits>
 
-namespace {
```

Comment:
anonymous namespace is prohibited in header files...
strictly speaking also the using, still the reality is that this file is included in only two files in this very plugin directory